includes.mk

```diff
@@ -10,7 +10,10 @@ SELF_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
 DOCKER_HOST = $(shell echo $$DOCKER_HOST)
 REGISTRY = $(shell echo $$DEV_REGISTRY)
 GIT_SHA = $(shell git rev-parse --short HEAD)
-IMAGE_PREFIX := deis/
```

Comment:
> IIRC you could overwrite this at runtime

Yes, and for our CI setup it's crucial that we set `IMAGE_PREFIX=deisci/` so the go in the correct org.